examples/2d/plot_sklearn.py

```diff
@@ -0,0 +1,43 @@
+"""
+ScikitLearn Transformer
+=====================================================================
+Here we demonstrate a simple application of scattering as a transformer
+"""
+
+
+from kymatio import Scattering2D
+from kymatio.utils import ScatteringTransformer
+from sklearn.pipeline import Pipeline
+from sklearn.model_selection import train_test_split
+from sklearn import datasets
+from sklearn.linear_model import LogisticRegression
+
+# Use numpy or torch for the scattering
+frontend = 'numpy'
+
+
+# Create a scattering object (can be Scattering1D or Scattering3D as well)
+S = Scattering2D(shape = (8, 8), J = 1, frontend = frontend)
```

Comment:
this example is not very realistic. consider a CIFAR-sized image with J>1

Reply:
also, i don't think that you need space around the equal sign for kwargs

Reply:
we have lots of CIFAR and MNIST examples, but these take a long time to run, I think for this one it make sense if its one of the clean and fast running examples

Reply:
I agree. It's better to have smaller examples in general.